zproject/prod_settings_template.py

```diff
@@ -662,3 +662,6 @@
 # Controls the Big Blue Button video call integration.  You must also
 # set big_blue_button_secret in zulip-secrets.conf.
 # BIG_BLUE_BUTTON_URL = "https://bbb.example.com/bigbluebutton/"
+
+# File where your GIPHY API key is stored
+GIPHY_SECRETS_FILE = '/etc/zulip/giphy.key'
```

Comment:
Are their API keys files or just strings?  For strings we usually use `get_secret` rather than a dedicated file.

Reply:
Just string, okay, added it to default_settings.py.